src/cards.csv

```diff
@@ -135,115 +135,111 @@
+"141","Why","Why does the client want Futurice to do this project?","a","1","","",""
 "140","Vision","What is the long-term vision of the service?","a","1","","",""
 "139","Release frequency","How frequent are production releases?","g","2","","",""
-"138","Product Owner","How long has the PO been working for the client organization?","g","3","","",""
-"137","SW buyer","Has the client a long track record of buying software?","a","1","","",""
-"136","Dependencies","Can we easily communicate with technical people of services we depend on?","g","2","","",""
-"135","PO time","How much time does the PO have available for the project?","g","3","","",""
-"134","Assets","How do we get client assets?","g","2","Think about texts, logo's brand material etc.","",""
+"138","Product Owner","How long has the Product Owner been working for the client organisation?","g","3","","",""
+"137","SW buyer","Does the client have a long track record of buying software?","a","1","","",""
+"136","Dependencies","Can we easily communicate with any 3rd party services we depend on?","g","2","","",""
+"135","PO time","How much time does the Product Owner have available for the project?","g","3","","",""
+"134","Assets","How can we access client assets like fonts, logos, brand material, research etc?","g","2","Think about texts, logo's brand material etc.","",""
 "133","Previous projects","What other projects have we done for the client and what were the biggest problems?","g","1","","",""
-"132","Approval","Who approves the final products?","a","3","","",""
-"131","Key players","Who are the key players in the client organization?","a","3","","",""
-"130","Agile","How agile is the client?","g","3","","",""
-"129","Digital maturity","What do we know of the client digital maturity?","a","1","","",""
-"128","Public reference","Is the case a public reference case?","a","2","","",""
+"132","Approval","Who approves the final deliverables of our product?","a","3","","",""
+"131","Key players","Who are the key players in the client organisation? Can we work without access to them?","a","3","","",""
+"130","Agile","Does the client understand Agile?","g","3","","",""
+"129","Digital maturity","What do we know about the client’s digital maturity?","a","1","","",""
+"128","Public reference","Is it possible to talk about the project publicly?","a","2","","",""
 "127","Feedback","How do we get client satisfaction feedback?","g","2","","",""
-"126","End users","Who are the end users?","g","1","","",""
-"125","Happy client","What is an outcome that would make the client jump with joy?","a","3","","",""
-"124","ROI","How does the client get the invested money back?","a","1","","",""
+"126","End users","Who are the likely end users of our service?","g","1","","",""
+"125","Happy client","What would make the client jump for joy during this project?","a","3","","",""
```

Comment:
This changes the focus from outcome to execution. Maybe remove the part after joy?

Reply:
👍